apps/server/src/modules/task/repo/task.repo.integration.spec.ts

```diff
@@ -4,7 +4,16 @@ import { MikroOrmModule } from '@mikro-orm/nestjs';
 import { MikroORM } from '@mikro-orm/core';
 import { MongoMemoryServer } from 'mongodb-memory-server';
 import { TaskRepo } from './task.repo';
-import { CourseTaskInfo, FileTaskInfo, LessonTaskInfo, Submission, Task, UserTaskInfo } from '../entity';
+import {
+	CourseGroupInfo,
```

Comment:
we should reduce the amount of info-entities by using entities one by collection and reduce the published fields finally in the api mappers

Reply:
this is one entity per collection...

I named it info entity for now because the module the entity should live in eventually doesnt exist yet.